hrcase/xml/case/Exiting.xml

```diff
@@ -59,20 +58,22 @@
         </ActivityType>
```

Comment:
I just noticed that, different from the wiki page, the ticket description says it should be 0. So I don't know what value it should actually be